src/Client/Suggest.cpp

```diff
@@ -133,8 +164,17 @@ void Suggest::loadImpl(Connection & connection, const ConnectionTimeouts & timeo
     }
 
     query << ") WHERE notEmpty(res)";
+    return query.str();
+}
 
-    fetch(connection, timeouts, query.str());
+void Suggest::loadImpl(Connection & connection, const ConnectionTimeouts & timeouts, size_t suggestion_limit)
+{
+    fetch(connection, timeouts, getLoadSuggestionQuery(suggestion_limit));
+}
+
+void Suggest::loadImpl(ContextMutablePtr context, size_t suggestion_limit)
+{
+    executeQuery(getLoadSuggestionQuery(suggestion_limit), context);
```

Comment:
But you don't read any data, so completion will not work (only basic words).

FWIW you may want to add `clickhouse-local` into the `01676_long_clickhouse_client_autocomplete` to cover it